SpatialGDK/Source/SpatialGDKTests/SpatialGDK/Interop/Connection/SpatialConnectionManagerTest.cpp

```diff
@@ -54,6 +54,26 @@ CONNECTIONMANAGER_TEST(SetupFromURL_Locator_CustomLocator)
 	return true;
 }
 
+CONNECTIONMANAGER_TEST(SetupFromURL_Locator_CustomLocator_CustomPort)
+{
+	// GIVEN
+	FTemporaryCommandLine TemporaryCommandLine("");
+	const FURL URL(nullptr, TEXT("10.20.30.40?locator?customLocator?customPort=7000?playeridentity=foo?login=bar"), TRAVEL_Absolute);
```

Comment:
Think it's worth having a test for the "ignoring URL port" case?

Reply:
Aye exactly, wondering if we want to make a test to prevent someone from implementing it in the future xD

Reply:
Yep, that's a good point. I'll add one 👍